nuclear_simulator/data_gen/config_engine/templates/nuclear_plant_comprehensive_config.yaml

```
@@ -0,0 +1,1190 @@
+          threshold: 0.96              # 96% of design flow
+          comparison: "less_than"
+          action: "pump_inspection"
+          cooldown_hours: 720.0        # Monthly cooldown
+          priority: "MEDIUM"
+        pump_power:
+          threshold: 1.05              # 105% of design power
+          comparison: "greater_than"
+          action: "pump_inspection"
+          cooldown_hours: 720.0        # Monthly cooldown
+          priority: "MEDIUM"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)